### PR TITLE
CPU frequency module

### PR DESCRIPTION
**Pull request by gacekjk**
It shows cpu frequency based on /proc/cpuinfo.
Basic tests included.

### CODE REVIEW

File: i3pystatus/cpu_freq.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+"""
+The module gathers information by default from `/proc/cpuinfo` about the current cpu frequency
+"""
+# coding=utf-8
```
Comment (enkore) [inline i3pystatus/cpu_freq.py]: Not necessary with Python 3

File: i3pystatus/cpu_freq.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+"""
+The module gathers information by default from `/proc/cpuinfo` about the current cpu frequency
```
Comment (enkore) [inline i3pystatus/cpu_freq.py]: This docstring should go into the class docstring below ; the docstring of Python modules is ignored.

File: i3pystatus/cpu_freq.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+class CpuFreq(IntervalModule):
+    """
+    class uses by default `/proc/cpuinfo` to determine the current cpu frequency
+
+    .. rubric:: Available formatters
+
+    * `{avg}` - mean from all cores in MHz `4.3f`
+    * `{avgg}` - mean from all cores in GHz `1.2f`
+    * `{corex}` - frequency of a selected core in MHz `4.3f`
+    * `{corexg}` - frequesncy of a selscted core in GHz `1.2f`
```
Comment (enkore) [inline i3pystatus/cpu_freq.py]: It's not very clear from the description that `x` is just a placeholder here, maybe rephrase it into `{coreX} - frequency of core X in ...` or so

**Comment by enkore** [comment]
Just some minor things, looks good.

**Comment by gacekjk** [comment]
Hopefully right now it's better :smile:
